.github/workflows/playwright.yml

```diff
@@ -1,23 +1,27 @@
 name: Playwright Tests
 on: [push, pull_request]
+
+env: 
+  YEXT_API_KEY: ${{ secrets.YEXT_API_KEY }}
+
 jobs:
-  test:
+  test-e2e:
     timeout-minutes: 5
     runs-on: ubuntu-latest
+    container:
+      image: mcr.microsoft.com/playwright:v1.38.0-jammy
     steps:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
       with:
         node-version: 18
     - name: Install dependencies
-      run: npm ci
-    - name: Install Playwright Browsers
-      run: npx playwright install --with-deps
+      run: npm ci && npm i --prefix ./test-site
+    - name: build package
+      run: npm run build
     - name: Run Playwright tests
+      env:
+        # Refer to: https://github.com/microsoft/playwright/issues/6500#issuecomment-1577221956
+        HOME: /root
       run: npx playwright test
-    - uses: actions/upload-artifact@v3
```

Comment:
Shouldn't we keep this? I thought the point of this was the save the test demo of each run.

Reply:
It was not doing anything as we do not generate a playwright-report directory on run of the playwright tests, so I removed. We have it set to print out the report in the logs, or generate an html page if running locally. I do not think having an artifact will be that useful, and will just slow the tests down.